dev-system/src/cmr/dev_system/config.clj

```diff
@@ -24,11 +24,16 @@
   {:default 10000
    :type Long})
 
-(defconfig embedded-kibana-port
+(defconfig embedded-kibana-gran-port
   "Specifies port to run an embedded kibana on."
   {:default 5601
    :type Long})
 
+(defconfig embedded-kibana-non-gran-port
```

Comment:
Should we rename this to just embedded-kibana-port?  Some point really soon, - when Tyler merges in his changes, we will only need 1 kibana instance because the monitoring cluster will have it for all clusters.